docs/data/joy/getting-started/quick-start/quick-start.md

```diff
@@ -0,0 +1,67 @@
+# Quick start
```

Comment:
I'm wondering if we shouldn't call this the day-to-day "Get started" page? Then, we can naturally continue to tailor the message as in "Joy is easy to get started with".

Reply:
Thoughts about that? @siriwatknp